src/library/Hooks/useInflation/index.tsx

```diff
@@ -35,20 +38,24 @@ export const useInflation = () => {
       stakeTarget -
       Math.min(auctionMax, numAuctions.toNumber()) * auctionAdjust;
     const idealInterest = maxInflation / idealStake;
-    const inflation =
-      100 *
-      (minInflation +
-        (stakedFraction <= idealStake
-          ? stakedFraction * (idealInterest - minInflation / idealStake)
-          : (idealInterest * idealStake - minInflation) *
-            2 ** ((idealStake - stakedFraction) / falloff)));
+
+    const totalIssuanceInTokens = totalIssuance
+      .div(BN_BILLION)
+      .div(BN_THOUSAND);
+
+    const inflation = totalIssuanceInTokens.isZero()
+      ? 0
+      : 100 * (yearlyInflationInTokens / totalIssuanceInTokens.toNumber());
+
+    let stakedReturn = stakedFraction ? inflation / stakedFraction : 0;
```

Comment:
Not related to this file perhaps, but min nominator bond is displayed incorrect value when connected to mainnet

![image](https://user-images.githubusercontent.com/3909333/204995077-249eaacb-bf7b-4e9d-8d53-814904f4a0fc.png)